blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,286 @@
+# common
+common.date.duration.format=d[d] h[h] m[m] s[s]
+common.date.duration.hint.format=M [month], d [days], h[ hours], m[ minutes], s[ seconds]
+common.date.duration.display.format=d[d] h[h] m[m] s[s]
+common.date.readable.long=MMM DD YYYY h:mma Z
+common.date.readable.short=MMM DD h:mma Z
+#加载中...
+common.pager.loading=\u52a0\u8f7d\u4e2d\u002e\u002e\u002e
```

Comment:
@tiansworld I can not guarantee this, as they seem to be handsynced? Maybe @meiboyu can explain more about his method of work.

Besides, is the Unicode encoding really necessary... I have noticed more Java projects do this. Haven't used this language for years now, so kinda lost touch with it... but since they still use Maven, it does not surprise me it is likely a necessity. Maybe someone can indulge me...